docs/en/latest/discovery/consul.md

```diff
@@ -0,0 +1,296 @@
+---
+title: consul
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Summary
+
+APACHE APISIX supports Consul as a service discovery
+
+## Configuration for discovery client
+
+### Configuration for Consul
+
+First of all, we need to add following configuration in `conf/config.yaml` :
+
+```yaml
+discovery:
+  consul:
+    servers:
+      - "http://127.0.0.1:8500"
+      - "http://127.0.0.1:8600"
```

Comment:
8500 and 8600 are different consul DNS servers that provide different query information?

Reply:
they are different clusters, i add some comment in consul.md